### ISSUE TITLE
Make loading message more user-friendly

### ISSUE THREAD

**Issue by cmiya** [cwrc/HuViz#18]
Problem:
- Currently, when you load a new dataset a message pops up that reads "feching" and then "parsed relation" with a count of the number (EX: Parsed relation 400). Previous suggestion was to change "Parsed quad 400" to "Loaded relationship 400."
- BUT what significance does this information hold for the user? Is this really information the user needs to know?

Solution:
- Instead of displaying a message with the parsed relation, display a percentage. For example, loading 20% etc. That way, the user can have a better understanding of the amount of time until completion.

**Comment by ilovan** [comment]
Agreed with @cmiya . A percentile progress bar would provide more information to the user and a message with the total number of relationships could be displayed after the dataset is completely loaded.

**Comment by cmiya** [comment]
The pre-loaded datasets still read fetching/parsed. The Custom datasets sometimes take awhile to load (at least when using the latest upToDateTestData.ttl file), but no message of any kind appears.
